src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -29,6 +25,9 @@ class RealtimeLinkingController(
     @Inject
     private lateinit var authz: AuthorizationManager
 
+    @Inject
+    private lateinit var backgroundLinkingService: BackgroundLinkingService
```

Comment:
I don't think the RealtimeLinkingController needs the BackgroundLinkingService

Reply:
I am adding some APIs for controlling linking.